modules/eco-create-nodemaintenance-cr-web-console.adoc

```diff
@@ -0,0 +1,30 @@
+// Module included in the following assemblies:
+//
+//nodes/nodes/eco-node-maintenance-operator.adoc
+
+:_content-type: PROCEDURE
+[id="eco-create-nodemaintenance-cr-web-console_{context}"]
+= Creating the NodeMaintenance custom resource in the web console
+
+You can create a `NodeMaintenance` custom resource (CR) from the web console
+
+.Prerequisites
+
+* Log in as a user with `cluster-admin` privileges.
+* Install the Node Maintenance Operator from the *OperatorHub*.
+
+.Procedure
+
+. From the *Administrator* perspective in the web console, navigate to *Operators* → *Installed Operators*.
+
+. Select the Node Maintenance Operator from the list of operators.
+
+. In the *Node Maintenance* tab, click *Create NodeMaintenance*.
+
+. In the *Create NodeMaintenance* page, select to configure the `NodeMaintenance` CR via *Form view* or *YAML view*.
+
+. To create the `NodeMaintenance` CR from via *Form view* or *YAML view*, click *Create*.
```

Comment:
Can this be combined with the previous step? I'm assuming the process is that the user selects the CR via the Form/YAML view and clicks Create? If so, maybe something like this?
> . To configure the `NodeMaintenance` CR from the **Create NodeMaintenance** page, select `NodeMaintenance` via the **Form view** or **YAML view** and click **Create**.